pom.xml

```diff
@@ -51,6 +51,8 @@
 ]]>
     </checkstyle.header.template>
     <findbugs.failOnViolation>true</findbugs.failOnViolation>
+    <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
```

Comment:
See https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L178 and https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L573